src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Animal {
+
+    public void eat() {
```

Comment:
we can leave it abstract, so subclasses would need to override method obligatory